src/nunjucks/moduk/components/breadcrumbs/__examples__/default.njk

```diff
@@ -0,0 +1,18 @@
+{%- from "moduk/components/breadcrumbs/macro.njk" import modukBreadcrumbs -%}
+
+{{ modukBreadcrumbs({
+  items: [
+    {
+      text: "Home",
+      href: "#"
+    },
+    {
+      text: "Passports, travel and living abroad",
+      href: "#"
+    },
+    {
+      text: "Travel abroad",
+      href: "#"
+    }
```

Comment:
I think it would be good to make the example less GOV.UK sounding. Maybe we can just use `Home > Components > Breadcrumbs` like https://storybook.design-system.digital.mod.uk/?path=/docs/breadcrumbs--default (for now at least)